css/index.css

```diff
@@ -102,11 +102,43 @@ p {
     padding-bottom: 20px;  
 }
 
+.top-content-a {
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+}
+
+.top-content-a .text-container-a {
+    margin: 0px 15px;
+    padding: 1%;
+    padding-bottom: 20px;  
+}
+
 .middle-content {
     margin-bottom: 20px;
     border-bottom: 1px dashed black;
 }
 
```

Comment:
You could possibly drop the middle-about in front of mid-box and the button selectors and still get the styles to work.